package/gluon-mesh-batman-adv-brmldproxy/files/etc/hotplug.d/iface/50-gluon-mesh-batman-adv-brmldproxy

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+([ "$INTERFACE" != "bat0" ] || [ "$ACTION" != "ifup" ]) && exit 0
+
+[ "$(gluon-show-site | jsonfilter -e "@.mesh.filter_membership_reports")" = "false" ] && exit 0
```

Comment:
Printing the whole site and then filtering it inefficient. You can copy the `lookup_site` helper from `package/gluon-mesh-batman-adv/files/lib/netifd/proto/gluon_bat0.sh`.